client/src/components/modals/nr-not-required.vue

```diff
@@ -45,3 +52,12 @@ export default class NrNotRequired extends Vue {
 }
 
 </script>
+
+<style lang="scss" scoped>
+  #nr-required-close-btn {
+    width: 6.375rem;
+  }
+  a {
+    text-decoration: underline;
+  }
```

Comment:
Very minor, an extra indent here. I think in this project most of the scss start on the same indent as the style tag.